Sources/Networking/Core/Upload/MultipartFormData+BodyPart.swift

```diff
@@ -7,9 +7,11 @@
 
 import Foundation
 
+extension InputStream: @unchecked @retroactive Sendable {}
```

Comment:
this one is tricky. It looks like `InputStream` won't become sendable in the future according to this developer thread https://developer.apple.com/forums/thread/757433 😢 
But I would address this later, as with other `@unchecked Sendables` due to API updates from Apple 😄